web/src/main/java/com/bazaarvoice/emodb/web/scanner/ScanUploader.java

```diff
@@ -193,6 +200,7 @@ private void startScanUpload(String scanId, ScanStatus status, Set<String> place
             long expireTime = compactionControlTime + Duration.ofHours(10).toMillis();
             // Update the scan start time in Zookeeper in all data centers.
             _compactionControlSource.updateStashTime(scanId, compactionControlTime, Lists.newArrayList(status.getOptions().getPlacements()), expireTime, _dataCenters.getSelf().getName());
+            _compactionControlTimeCounter.inc();
```

Comment:
Should this (and the corresponding call to `dec()`) be here?  If I remember correctly the compaction control time gets set in both data centers by the delegate but this counter only would track local calls to increment and decrement.  This is ok, but if that's the intent then maybe the metric name should reflect that it only is counting locally started stashes.  Otherwise I'd recommend moving this metric to `DefaultCompactionControlSource`.